stories/core/ComboBox.stories.tsx

```diff
@@ -387,3 +389,30 @@ export const WithLabel: Story<Partial<ComboBoxProps<string>>> = (args) => {
 WithLabel.args = {
   inputProps: { id: 'combo-input', placeholder: 'Select a country' },
 } as ComboBoxProps<string>;
+
+export const WithStatus: Story<Partial<ComboBoxProps<string>>> = (args) => {
+  const options = React.useMemo(() => countriesList, []);
+
+  return (
+    <ComboBox
+      options={options}
+      inputProps={{ placeholder: 'Select a country' }}
+      onChange={(value: string) => action(value ?? '')()}
+      status='negative'
+      message={
+        <div style={{ display: 'flex', gap: 4, alignItems: 'center' }}>
```

Comment:
So you still need to add icon and style it yourself??
Naah, I prefer `InputGroup` then..

Reply:
It's on you if you want to use the InputGroup hack to avoid writing two lines of CSS, but the `status` styling is useful even without `message`

Reply:
Hack is wrong word. I think it is nice universal approach, just our css is buggy sometimes..

Reply:
I would still call it a hack because you are forcing it in situations it was not designed for. And it's also bad for accessibility as there is no `label` associated with the input.
![image](https://user-images.githubusercontent.com/9084735/145451233-3ed2817a-0602-4297-ba21-0de6b8cd1409.png)

Reply:
I would agree with Vykintas that `InputGroup` looks better. And it is not jus 2 lines of CSS, it is all 5 CSS rules. We are the ones that should define standards and not leave them for the user. We might also change something in the inputs, this message styling will differ from the other inputs.

Reply:
But we do need the status styling on the input, which you do not get when using InputGroup... so what to do? Applying twice status also not  that nice :(